packages/react-reconciler/src/ReactFiberCommitWork.js

```diff
@@ -756,6 +756,12 @@ function commitUnmount(current: Fiber): void {
       }
     }
   }
+
+  // Remove reference for GC
+  current.stateNode = null;
+  if (current.alternate != null) {
```

Comment:
Should this be `if (current.alternate !== null) {`?

Reply:
Probably because of this:
```
> const a = {}
undefined
> a.x != null
false
> a.x == null
true
> a.x === null
false
```

It's a nice way to cover `null` and `undefined` scenarios.